<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="meedo74-ThinkPad-X230" failures="1" tests="1" name="comparing item " time="4.005" errors="0" timestamp="2024-01-22T00:08:48 EET">
  <testcase classname="testcases.T03_item_action" name="checking_item_header_name" time="1.179">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Sauce Labs Fleece Jacket] but found [ITEM NOT FOUND]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Sauce Labs Fleece Jacket] but found [ITEM NOT FOUND]
at testcases.T03_item_action.checking_item_header_name(T03_item_action.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- checking_item_header_name -->
</testsuite> <!-- comparing item  -->
